docs/resources/project.md

```diff
@@ -60,33 +57,16 @@ resource "foundry_project" "example-project" {
 ### Required
 
 - `display_name` (String) Display name of the Project.
-- `organizations` (Set of String) List of Organizations applied to this Project.
 - `space_rid` (String) RID of the Space that this Project belongs to.
 
 ### Optional
 
 - `description` (String) Description of the Project.
-- `markings` (Set of String) List of Markings applied to this Project.
-- `resource_roles` (Set of Object) Set of Role Grants applied to this Project. (see [below for nested schema](#nestedatt--resource_roles))
+- `initial_default_roles` (Set of String) The initial Default Roles to be applied when creating the Project. Any changes to this field after Project creation will not be applied; instead, use the project_resource_roles resource to manage Default Roles using the'everyone' principal.
+- `initial_organizations` (Set of String) The initial list of Organizations to be applied when creating the Project. Any changes to this field after Project creation will not be applied; instead, use the project_organizations resource to manage Organizations.
+- `initial_resource_roles` (Map of List of Object) The initial set of Resource Roles to be applied when creating the Project. Any changes to this field after Project creation will not be applied; instead, use the project_resource_roles resource to manage Resource Roles.
```

Comment:
Why is `initial_default_roles` different from `initial_resource_roles` here? the separate project role resource doesn't make this distinction.